test/unit/lib/fetchers/PortionFetcher.test.ts

```diff
@@ -89,22 +91,24 @@ describe('PortionFetcher Unit Tests', () => {
     );
     expect(portionData.hasPortion).toEqual(GET_NO_PORTION_RESPONSE.hasPortion);
 
-    const cachedPortionData = portionCache.get<GetPortionResponse>(
-      PORTION_CACHE_KEY(tokenInChainId, tokenOutChainId, tokenInAddress, tokenOutAddress)
-    );
-    expect(cachedPortionData).toBeDefined;
-    expect(cachedPortionData?.hasPortion).toEqual(GET_NO_PORTION_RESPONSE.hasPortion);
+    if (!forcePortion) {
```

Comment:
nit - asserting no negative caching may be desired unit test coverage, because test header is designed to bypass the negative cache. not a blocker.

Reply:
true